### PR TITLE
Fix/add current uesr to user controller

### PR DESCRIPTION
**Pull request by VladFomenko**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -10,6 +10,6 @@ class ApplicationController < ActionController::API
   private
 
   def current_user
-    User.find(@decoded[:user_id]) if @decoded.present?
+    User.find_by(id: @decoded[:user_id]) if @decoded.present?
```
Comment (oleksiimikhno) [inline app/controllers/application_controller.rb:13]: find always used to field
```ruby
:id 
``` 
, why you change logic to
```ruby
find_by 
``` 
 for a specific field? 
 https://medium.com/@mizukilee/useful-active-record-methods-where-find-find-by-and-order-6890e598553e
 https://medium.com/@abhishektanwar9596/find-vs-find-by-in-rails-d319505a7b3

File: spec/requests/api/v1/users_spec.rb [hunk 2]
```diff
@@ -5,23 +5,6 @@
   let(:Authorization) { "Bearer #{generate_jwt_token(user)}" }
 
   path '/api/v1/users' do
-    get('list users') do
```
Comment (oleksiimikhno) [inline spec/requests/api/v1/users_spec.rb:8]: I think this method we need for a blog (maybe for a mention) or QA

Comment (VladFomenko) [reply spec/requests/api/v1/users_spec.rb:8]: Now we don't have this functional.

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (VladFomenko) [reply app/controllers/application_controller.rb:13]: I had a problem, when I deleted user, current user method tried to find user which I deleted and I got a problem like 404 error. When I used find_by I received nil, and it works in my case.

**Review by oleksiimikhno — APPROVED** [review]

**Pull request merged**